aptos-move/aptos-vm/README.md

```diff
@@ -1,101 +1,32 @@
-   result of executing the bytecode, the changes are not applied to the global
-   blockchain state by the VM---this is the responsibility of the
-   [execution module](../../../execution/).
-3. ***Transaction Epilogue*** - in execution mode the epilogue defined in
-   the [Aptos account module](../framework/modules/diem_account.mvir) is
-   executed to perform actions based upon the result of the execution of
-   the user-submitted transaction. One example of such an action is
-   debiting the gas fee for the transaction from the submitting account's
-   balance.
-
-During execution, the runtime resolves references to code by loading the
-referenced code via the data view. One can think of this process as similar
-to linking. Then, within the context of a block of transactions---a list of
-transactions coupled with a data view---the runtime caches code and
-linked and imported modules across transactions within the block.
-The runtime tracks state changes (data updates) from one transaction
-to the next within each block of transactions; the semantics of the
-execution of a block specify that transactions are sequentially executed
-and, as a consequence, state changes of previous transactions must be
-visible to subsequent transactions within each block.
-
-## Implementation Details
-
-* The runtime top level structs are in `runtime` and `diem vm` related
-  code.
-* The transaction flow is implemented in the [`process_txn`](./src/process_txn.rs)
-  module.
-* Code caching logic and policies are defined under the [code
-  cache](../../move-vm/runtime/src/code_cache/) directory.
-* Runtime loaded code and the type system view for the runtime is defined
-  under the [loaded data](src/loaded_data/) directory.
-* The data representation of values, and logic for write set generation can
-  be found under the [value](./src/value.rs) and [data
-  cache](./src/data_cache.rs) files.
-
-## Folder Structure
-
-```
-.
-├── src                 # VM Runtime files
-│   ├── code_cache      # VM Runtime code cache
-│   ├── loaded_data     # VM Runtime loaded data types, runtime caches over code
-│   ├── unit_tests      # unit tests
-├── vm-cache-map        # abstractions for the code cache
-```
-
-## This Module Interacts With
-
-This crate is mainly used in two parts: AC and mempool use it to determine
-if it should accept a transaction or not; the Executor runs the MoveVM
-runtime to execute the program field in a SignedTransaction and convert
-it into a TransactionOutput, which contains a writeset that the
-executor need to patch to the blockchain as a side effect of this
-transaction.
+### AptosVM
+
+AptosVM is used to execute a single Aptos transaction.
+Executes both system and user transactions.
+For every user transaction, the following steps are performed:
+
+1. ***Prologue:***
+   Responsible for checking the structure of the transaction such as signature verification, balance checks on the account paying gas for this transaction, etc.
```

Comment:
In the previous version of this document, it is mentioned that bytecode verifier runs in the prologue in verifier mode.

In this new text, the bytecode verifier is not discussed. Perhaps add a brief discussion?

Reply:
It doesn't necessarily run in prologue, so the old text is wrong (and is probably very old - there used to be no entry functions and scripts were the only way to run code iirc). Bytecode verifier runs for scripts, still, but for entry functions, and this is part of step 2. I added a few sentences explaining this.